src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -33,34 +28,23 @@ public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
         Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
+
+        List<StationResponse> stations = StationResponse.of(persistLine.getStations());
         return LineResponse.of(persistLine, stations);
     }
 
     public List<LineResponse> findLines() {
```

Comment:
```suggestion
    @Transactional(readOnly = true)
    public List<LineResponse> findLines() {
```

조회만을 하는 경우 readOlny 옵션을 사용할 수 있을 것 같아요

Reply:
네!